### ISSUE TITLE
No depth information shown 

### ISSUE THREAD

**Issue by bob0the0mighty** [drewnoakes/metadata-extractor-dotnet#272]
I can use exiftool to display/export depth information contained in a image, but this skips over it.

I'm not sure if this should be a feature request or if this is outside the scope of this project. 

Please let me know which and I can either add more info or close this issue.

**Comment by drewnoakes** [comment]
Yes, this is a feature request and it's something we would consider adding to the library for sure.

What device is capturing this data? Can you provide sample images for testing? Would you like to work on adding this feature?

**Comment by bob0the0mighty** [comment]
I wouldn't mind trying to add it, but it seems that it's implemented differently for different phones. 

I'm working with pictures from a pixel 2 and pixel 4a.
Exiftool returns values for pictures taken with the pixel 2, but nothing for the pixel 4a. 
PhotoPea shows depth info for both.
Gimp shows nothing for either.

I should have an IPhone to test soon, as well.

I'd appreciate any direction on where to begin.

**Comment by drewnoakes** [comment]
I am not surprised that there may be more than one implementation of depth data.

The best place to start would be with some kind of specification, or another implementation that has a compatible license with Apache 2.0.

If the files are JPEG, then you'd need to identify which JPEG segment type contains the depth information. There's likely some preamble to that data too, to differentiate it from other kinds of data that can be found in the same segment.

The discussion on this issue might be helpful. It applies to the Java library, and a different type of camera, but there's some functional overlap I'm sure: https://github.com/drewnoakes/metadata-extractor/issues/197
